solr/solrj/src/java/org/apache/solr/client/solrj/impl/CloudHttp2SolrClient.java

```diff
@@ -461,5 +458,33 @@ public CloudHttp2SolrClient build() {
 
       return new CloudHttp2SolrClient(this);
     }
+
+    protected HttpSolrClientBase createOrGetHttpClient() {
+      if (httpClient != null) {
+        return httpClient;
+      } else if (internalClientBuilder != null) {
+        return internalClientBuilder.build();
+      }
+
+      HttpSolrClientBuilderBase<?, ?> builder;
+      if (HTTP_JETTY_SOLR_CLIENT_BUILDER_CTOR != null) {
+        try {
+          log.debug("Using Http2SolrClient as the delegate http client");
+          builder = HTTP_JETTY_SOLR_CLIENT_BUILDER_CTOR.newInstance();
+        } catch (RuntimeException e) {
+          throw e;
+        } catch (Exception e) {
+          throw new RuntimeException(e);
+        }
+      } else {
+        log.debug("Using HttpJdkSolrClient as the delegate http client");
+        builder = new HttpJdkSolrClient.Builder();
+      }
+      return builder.build();
+    }
+
+    protected LBSolrClient createOrGetLbClient(HttpSolrClientBase myClient) {
+      return myClient.getLBSolrClient();
```

Comment:
This is why I added getLBSolrClient to the base.  This way Jetty & JDK impls can do their own thing.